great_expectations/data_context/data_context.py

```diff
@@ -881,7 +883,15 @@ def checkpoint_store(self):
             )
 
     @property
-    def expectations_store_name(self):
+    def profiler_store_name(self) -> str:
+        return self.project_config_with_variables_substituted.profiler_store_name
+
+    @property
+    def profiler_store(self) -> ProfilerStore:
+        return self.stores[self.profiler_store_name]
```

Comment:
May need to be more robust but should work for now

Reply:
@cdkini Either now, or in a very near PR (would need a JIRA task), could we please return a default profiler store much like we do for `Checkpoint` -- in case the configuration does not have it.  This involves some exception handling and returning the default profiler store (and default profiler store name), respectively, depending on the method call.

Reply:
@cdkini Ideally now, because users with current configuration will get an exception...

Reply:
done!